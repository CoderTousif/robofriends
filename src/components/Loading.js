import React from "react";
// import Typography from '@material-ui/core/Typography';
// import { makeStyles } from '@material-ui/core/styles';
import CircularProgress from "@material-ui/core/CircularProgress";

// const useStyles = makeStyles((theme) => ({
//     root: {
//         display: 'flex',
//         '& > * + *': {
//             marginLeft: theme.spacing(2),
//         },
//     },
// }));

function Loading() {
    // const classes = useStyles();
    return (
        <div
            style={{
                display: "flex",
                justifyContent: "center",
                alignSelf: "center",
                height: "50%",
            }}
        >
            <CircularProgress color="secondary" />
        </div>
    );
}

export default Loading;
