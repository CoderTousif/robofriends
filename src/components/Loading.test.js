import React from "react";
import Loading from "./Loading";
import { shallow } from "enzyme";

it("should render loading component", () => {
    expect(shallow(<Loading />)).toMatchSnapshot();
});
