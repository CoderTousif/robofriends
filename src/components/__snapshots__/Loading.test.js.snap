// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loading component 1`] = `ShallowWrapper {}`;
